payara-managed/src/main/java/fish/payara/arquillian/container/payara/managed/PayaraManagedDeployableContainer.java

```diff
@@ -122,7 +123,7 @@ public void stop() throws LifecycleException {
 
     @Override
     public ProtocolDescription getDefaultProtocol() {
-        return new ProtocolDescription("Servlet 6.0");
+        return new ProtocolDescription(ServletProtocol.PROTOCOL_NAME);
```

Comment:
PROTOCOL_NAME Constant value is `Servlet 5.0`: https://github.com/arquillian/arquillian-core/blob/master/protocols/servlet-jakarta/src/main/java/org/jboss/arquillian/protocol/servlet5/v_5/ServletProtocol.java#L28

Reply:
Yes,  there is no implementation of Servlet 6 based arquillian protocol. Having 6 just resulted in NPE.
This is protocol implementation we test with, should there be Servlet 6 specific one sometime, users can change it via arquillian.xml